datastore/gcloud/aio/datastore/datastore.py

```diff
@@ -120,11 +120,15 @@ def make_mutation(
         if operation == Operation.DELETE:
             return {operation.value: key.to_repr()}
 
+        mutation_properties = {}
+        for k, v in (properties or {}).items():
+            value = v if isinstance(v, cls.value_kind) else cls.value_kind(v)
+            mutation_properties[k] = value.to_repr()
+
         return {
```

Comment:
Eventually I'd like to deprecate this method and have the logic inside the Mutation object. But I figured that should be a separate change.